apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -845,9 +845,13 @@ events:
         This event only needs to be used if your app is using the `urlSession` module to download data.
     properties:
       - name: taskIdentifier
-        summary: The `urlSession` download task's identifer.
+        summary: The `urlSession` download task's identifier.
         type: Number
 
+      - name: sessionIdentifier
+        summary: The `urlSession` session identifier. If it does not exist, this property is not provided.
+        type: String
```

Comment:
Yep, the problem is, that event properties cannot have the `since` property. I added it to the summary.